src/MakeListingDisplay/components/MakeListingDisplay.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import InputField from './InputField';
+
+import '../styles/MakeListingDisplay.css';
+
+/**Component class that displays the input fields for a user to post a listing */
+class MakeListingDisplay extends Component {
```

Comment:
I don't see any client side validation here. What happens if user does not enters anything for some fields? Do we set default values at server side? What about the date validations? End date optional ? Also end date should be > start date.

Reply:
All of the inputs have the required attribute, which means that they can't submit the form without filling in the information. I've added default values for the number inputs, so if someone has a single room but not a shared room, those fields will auto populate with 0. The next pr will have more robust validation like end date > start date because it will include storing all inputs in the state of the component